content/en/docs/armory-agent/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Armory Agent for Kubernetes"
+linkTitle: "Armory Agent for K8s"
+description: >
+  A native Kubernetes service that enables high-scale deployments to Kubernetes using Spinnaker<sup>TM</sup>
+weight: 20
+---
+
+Armory Agent is a new, flexible way for Spinnaker to interact with your Kubernetes infrastructure that solves the scaling issues plaguing Spinnaker's Kubernetes V2 Provider.
```

Comment:
I'd remove any negative reference to the V2 provider. (e.g. `issues plaguing...`). We're not building in opposition to the V2 provider, we're extending to offer more usage (account management flexibility, greater scale). The v2 provider is getting better on its own and it cannot be the agent's only value prop.